examples/bundles/lenet_mnist/CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+set(LENET_MNIST_BUNDLE_DIR ${GLOW_BINARY_DIR}/examples/bundles/lenet_mnist)
+set(LENET_MNIST_GLOW_S3 "http://fb-glow-assets.s3.amazonaws.com/models/lenet_mnist")
+set(LENET_MNIST_NET_FILES predict_net.pbtxt predict_net.pb init_net.pb)
+set(MODEL_INPUT_NAME "data")
+set(IMAGES ${GLOW_SOURCE_DIR}/tests/images/mnist)
+set(RUNTIME_OUTPUT_DIRECTORY ${LENET_MNIST_BUNDLE_DIR})
+
+# Run Commands
+# ============
+# Regular
+add_custom_command(
+  OUTPUT
+    RunBundleCommand
+  COMMAND
+    LeNetMnistBundle ${IMAGES}/*.png
```

Comment:
I just noticed only processes the first of the items produced by expanding `${IMAGES}/*.png`. I made this mistake with the Resnet bundle migration too.

The reason I wrote the command like this is that I couldn't get a bash for-loop to work. cmake didn't expand `LeNetMnistBundle` when I replaced the `COMMAND` with `for ...`. I think it has something to do with the reference to the cmake target being in the middle of the command instead of the beginning?